docs/encoding_issues-howto.md

```diff
@@ -4,26 +4,27 @@ SPDX-FileCopyrightText: 2021-present Open Networking Foundation <info@opennetwor
 SPDX-License-Identifier: Apache-2.0
 -->
 
-## How to deal with decoding problems?
-When you implement your SW in ONF’s RIC you may face some issues related to the encoding/decoding of the information 
+# How to deal with encoding/decoding problems?
+When you integrate your SW with ONF’s RIC you may face some issues related to the encoding/decoding of the information 
 sent over the wire. Here is a summary of potential problems you may face.
 
 ### Incompatibility of APER encodings is the most common issue.
-APER encoding is specified in [ITU-T X.691](https://www.itu.int/ITU-T/studygroups/com17/languages/X.691-0207.pdf) and may seem a bit vague.
-Reference asn1c tool recommended by O-RAN is Nokia’s distribution of [asn1c](https://github.com/nokia/asn1c).
+APER encoding is specified in [ITU-T X.691](https://www.itu.int/ITU-T/studygroups/com17/languages/X.691-0207.pdf) and may seem to be a bit vague.
+In O-RAN, encoding/decoding schema is generated with asn1c tool, which generates a C code. Reference asn1c tool recommended by O-RAN is 
+Nokia’s distribution of [asn1c](https://github.com/nokia/asn1c).
 
 [Go APER library](https://github.com/onosproject/onos-lib-go/tree/master/pkg/asn1/aper) is a [fork](https://github.com/free5gc/aper) 
-of the free5gc project. It was significantly reworked and enhanced. It is fully compatible with Nokia’s asn1c tool. 
-It was proved with unit tests for E2AP and E2SMs (KPMv2, RC-PRE, MHO, Test-SM).
+of the free5gc project, which was significantly reworked and enhanced. It is fully compatible with Nokia’s asn1c tool (and thus 
```

Comment:
after `enhanced` you could could say `to work with protobuf driven Go structs, rather than plain Go structs`